### DISCUSSION TITLE
GPS fix

### DISCUSSION THREAD

**Discussion by MaximMalkov** [iNavFlight/inav#7737] [Q&A] [answered]
My quad is not able to arm because of "GPS fix" issue. Any suggestions what should I do?

**Comment by JulioCesarMatias** [comment, marked as answer]
CLI >> set nav_extra_arming_safety = OFF >> save

**Reply by MrD-RC** [reply]
`set nav_extra_arming_safety = OFF` something that should be removed from iNav.

**Reply by MaximMalkov** [reply]
Thank you, that helped!

**Reply by rts18** [reply]
> `set nav_extra_arming_safety = OFF` something that should be removed from iNav.

Isn't that a bit harsh? I find that command ideal for bench testing.
Its even useful if one wants a quick test flight and navigation modes aren't of concern.

**Reply by MrD-RC** [reply]
> > `set nav_extra_arming_safety = OFF` something that should be removed from iNav.
> 
> Isn't that a bit harsh? I find that command ideal for bench testing. Its even useful if one wants a quick test flight and navigation modes aren't of concern.

Not at all, because there is the allow_bypass option; which is perfect for bench testing. This means you can force the checks to be bypassed when testing on the bench. But there is no chance of forgetting to re-enable the checks for actual flights.

**Reply by rts18** [reply]
> > > `set nav_extra_arming_safety = OFF` something that should be removed from iNav.
> > 
> > 
> > Isn't that a bit harsh? I find that command ideal for bench testing. Its even useful if one wants a quick test flight and navigation modes aren't of concern.
> 
> Not at all, because there is the allow_bypass option; which is perfect for bench testing. This means you can force the checks to be bypassed when testing on the bench. But there is no chance of forgetting to re-enable the checks for actual flights.

Understood. I thought you were talking about the whole command, including Bypass.

**Reply by MrD-RC** [reply]
No, just remove the off because it is a bad idea. If people forget it’s off, they just arm, fly, flick RTH and find there’s no home point stored. Even worse, they have a failsafe.

**Reply by OptimusTi** [reply]
Just a suggestion: Force a "System Message" in the OSD when the user arms with the nav_extra_arming_safety setting is OFF?

**Comment by MrD-RC** [comment]
Find out why you aren't getting a GPS fix.

Are you indoors testing? Is the GPS working? You need to provide more information.

**Reply by MaximMalkov** [reply]
I just assembled the Quad and testing it indoors. Actually there are no satellites were found. GPS is working. Probably it prevents to "Arm".

**Reply by MrD-RC** [reply]
This is why it can’t arm. You need to get a GPS  3D fix on 6 satellites before you can arm. 

I would highly recommend turning nav_extra_arming_safety back on. Switch this off is the easiest way to have a fly away.
